lua/telescope/_extensions/dap.lua

```diff
@@ -165,6 +180,47 @@ local variables = function(opts)
   }):find()
 end
 
+
+local frames = function(opts)
+  opts = opts or {}
+  local session = require'dap'.session()
+
+  if not session or not session.stopped_thread_id then
+    print('Cannot move frame if not stopped')
+    return
+  end
+  local frames = session.threads[session.stopped_thread_id].frames
+
+  pickers.new(opts, {
+    prompt_title = 'Jump to frame',
+    finder    = finders.new_table {
+      results = frames,
+      entry_maker = function(frame)
+        return {
+          value = frame,
+          display = frame.name,
+          ordinal = frame.name,
+          filename = frame.source.path,
+          lnum = frame.line or 1,
+          col = frame.column or 0,
+        }
+      end,
+    },
+    sorter = sorters.get_generic_fuzzy_sorter(),
```

Comment:
We have a new way of doing this with respecting user settings. I've forgot to update it :]
```lua
sorter = conf.generic_sorter(opts),
-- and conf being
-- local conf = require('telescope.config').values
```